### ISSUE TITLE
SG Craft interdimensional travel

### ISSUE THREAD

**Issue by Liiw** [SpongePowered/SpongeForge#1492]
Hello!

I am using [SG Craft](https://minecraft.curseforge.com/projects/sg-craft) on a 1.10.2 Spongeforge server with numerous other mods. I have run into an issue when trying to access the Nether:

Upon building the nether gate, I managed to successfully return to the overworld using it. After a while, when I tried to return, the controller informed me that there is no gate on the address specified. I double checked the address and finally used a nether portal to get back to the nether SG. After that I had a friend dial in from his gate and enter the Nether. When he came through, I could not see him (nor could he see me). Furthermore, he could not activate pressure plates. After relogging to the server, he was visible again and the pressure plates worked for him. 

Upon returning to the Overworld, I noticed that I too could not step on pressure plates, but only after relogging to the server. Later I tried to dial the Nether gate again, while having the server console visible for me. After dialing the address, the controller told me that there is no such address. On the server console, it stated that it attempted to initialize dimension -93 which wasn't registered and thus it fell back to overworld. The attempted DIM ID changed when I tried another test gate I set up in the nether. This would indicate that Spongeforge is somehow interfering with SG Craft's gate address system when it comes to interdimensional dialing. Further testing indicated these issues present in The End. Traveling between Overworld gates is unaffected.

These problems did not occur when using the gates in an SP world, which led me to believe that SpongeForge is causing this.

- SpongeForge version: 1.10.2-2281-5.2.0-BETA-2300
         ALSO: SpongeBootstrap-0.2.0
- Forge version: 12.18.3.2281
- Java version: 1.8.0-121 
- Operating System:

 - [ ] Linux (Ubuntu, Debian, Fedora, etc)
 - [ ] macOS
 - [x] Windows Server 2012 R2 Standard 64-bit

- Plugins/Mods: 

I didn't know how to export the command results here so I generated a list from the mods folder contents via cmd:

[list.txt](https://github.com/SpongePowered/SpongeForge/files/980214/list.txt)

**What is the expected result?**

Successful dialing of a Nether (or anoher dimension) gate and after traveling to or from the Nether, maintaining my corporeal self (i.e. ability to activate pressure plates or telepads etc.)

**What is the current result?**

Not being able to dial interdimensional gates (except back to the Overworld) unless the dimensions in question aren't previously loaded by players already there. After interdimensional travel not being able to activate pressure plates, telepads etc or even unable to use the Stargate to travel away, until relogging.

[fml-server-latest.txt](https://github.com/SpongePowered/SpongeForge/files/980197/fml-server-latest.txt)
[latest.txt](https://github.com/SpongePowered/SpongeForge/files/980198/latest.txt)

**Comment by bloodmc** [comment]
@Liiw Update to latest SpongeForge and retest. Make sure to include logs.

**Comment by bloodmc** [comment]
No response in months, closing.

**Comment by Liiw** [comment]
Hello again!

Terribly sorry about the lack of responses on my behalf. Our server was down for a couple of months because of some disputes between the moderators, and I managed to get it up and running just recently.

However, the issue persists. We are now using "spongeforge-1.10.2-2281-5.2.0-BETA-2464" on the server and there still is issues with the missing SG-s in the Nether (while physically there, and chunkloaded, the server cannot find them unless someone is in the vicinity). I am uploading a log which has the issue. The gate being dialed was in the Nether, but the log would suggest otherwise ("Attempt made to initialize dimension id 644 which isn't registered!, falling back to overworld.")

[fml-server-3.txt](https://github.com/SpongePowered/SpongeForge/files/1275629/fml-server-3.txt)

Again, apologies for the huge delay in responding, I will be more responsive in the future. Also, I would happily use a newer version of Spongeforge, but for some reason, it won't start when in the mods folder (the server starts up with a GUI i.e. just forge). Is there something changed in the installation of Sponge?

**Comment by bloodmc** [comment]
@Liiw We do not support outdated builds, you need to update to latest SF in order to be supported. Provide a new log with latest builds.

**Comment by Liiw** [comment]
@bloodmc I will try to get the latest build working on the server soon. I will inform you then. Thanks for the reply!

**Comment by ryantheleach** [comment]
@Liiw Can you disable entity-activation-range under modules by settings entity-activation-range to false? I suspect that it might be related to the symptoms you describe, If this works, you can enable it again, set auto-populate to true (under entity-activation-range) and see if the sg-s populate in the config, once that's done you should be able to set your own values / disable them.

If it doesn't work we are back to square one.

**Comment by Liiw** [comment]
@bloodmc I tried to update to the latest version, but I am having trouble and I don't know what I am doing wrong. #1762 

@ryantheleach I disabled entity-activation-range and reloaded the global configurations. The problem remained. I restarted the server and tried again, but to no avail. I am uploading the latest server log:

[fml-server-latest.txt](https://github.com/SpongePowered/SpongeForge/files/1277194/fml-server-latest.txt)

**Comment by Liiw** [comment]
As stated in #1762 I managed to finally upgrade Spongeforge to the latest version. I fired up the server and tested it. I had to physically travel to the Nether SG to "initialize" it after the update or something. After that, it worked with no problems- the gate would dial from either end and I could use pressure plates on both sides. I even restarted the server just to see if the problem reapperas, it didn't. This issue is now solved.

@bloodmc and @ryantheleach thank you both for your advice!
